www/js/chart.js

```diff
@@ -169,6 +169,14 @@ define([
 			return series;
 		},
 
+        formatDashedSeries: function(stroke){
+            array.forEach(stroke, function(s){
+                s["style"] = "ShortDash";
```

Comment:
I guess this was the only part that didn't work?  @sachingrover211 said he will make a fix later for the styling, which is fine.